varint.js

```diff
@@ -87,3 +85,9 @@ function readUnsignedVarIntFrom(buffer, offset) {
     var got = offset - start;
     return ReadResult.shortError(got + 1, got, start, offset);
 }
+
+function countBits(n) {
```

Comment:
Awesome find. I challenge you to use a faster algorithm. The ratio of logs is O(1). Looks like this might have simply been an off-by-one, not a bad approach. Play with:

```js
for (var index = 0; index < 10000; index++) {
    var expected = index === 0 ? -1 : index.toString(2).length;
    var actual = index === 0 ? -1 : Math.ceil(Math.log(index + 1) / Math.log(2));
    if (actual !== expected) {
        console.log(index, expected, actual);
    }
}
```